src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -165,7 +165,7 @@ class ItemPage extends React.Component {
       : null;
     const publisher = record.publisher && record.publisher.length ?
       <Link
-        to={{ pathname: '/search', query: { q: `publisher:"${record.publisher[0]}"` } }}
+        to={{ pathname: '/search', query: { q: `filter[publisher]=${record.publisher[0]}` } }}
         title={`Make a new search for publisher: "${record.publisher[0]}"`}
         onClick={(e) => this.onClick(e, `publisher:"${record.publisher[0]}"`)}
```

Comment:
I think it's just this one and the contributorLiteral above that needs to be updated and that's it!